### PR TITLE
Portfolio: finish mobile version

### PR DESCRIPTION
**Pull request by david63011**
Understand how to parse a Figma design to create a UI.
Use Flexbox to place elements in the page.
Use CSS Grid to place elements in the page.
Build a personal portfolio site.

### CODE REVIEW

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @david63011 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

woohoo-youre-almost

## Highlights

- Have used the right GitHub flow ✅
- Meaningful summary for PR ✅
- Linter checks passed ✅
## Required Changes ♻️
- [ REQUIRED] Your Readme  is incomplete . Would you please use the Readme template provided by Microverse and customize it. This is important for documentation purposes

_Check the comments under the review for more required changes._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index2.html [hunk 1]
```diff
@@ -53,7 +53,187 @@
...
       </div>
-      \
     </section>
+
+    <div class="top-one">
+      <h1 class="work">My Recent</h1>
+      <h1 class="work-2">Works</h1>
+    </div>
+
+    <img class="Indicator" src="Indicator.png" alt="green line" />
```
[the remark this answers is not in the document]
Comment (mgmediaweb) [reply index2.html]: I kindly suggest following the previous reviewer's advice and using the Section tag to wrap each part of your site, cover, projects, about, etc. Also for accessibility, it is not recommended to use more than 1 H1 tag.

File: index2.html [hunk 2]
```diff
@@ -53,7 +53,187 @@
...
+        <div class="project-box">
+          <h1 class="project-name1">Multi-Post Stories</h1>
+          <h1 class="project-name-two">Gain+Glory</h1>
+        </div>
+        <div class="languages">
+          <div class="ruby">Ruby on Rails</div>
+          <div class="css">css</div>
+          <div class="js">JavScript</div>
+          <div class="html">HTML</div>
+        </div>
```
[the remark this answers is not in the document]
Comment (mgmediaweb) [reply index2.html]: Good job following the previous reviewer's suggestions.

File: index2.html [hunk 3]
```diff
@@ -53,7 +53,187 @@
...
+        <h1 class="languages2">Languages</h1>
+        <img class="globe" alt="globe picture" src="ball.svg" />
+      </div>
+      <div class="center-row">
+        <div class="java-language">Javascript</div>
+        <div class="ruby-language">Ruby</div>
+        <div class="html-language">HTML</div>
+        <div class="css-language">CSS</div>
+      </div>
+    </div>
```
[the remark this answers is not in the document]
Comment (mgmediaweb) [reply index2.html]: Good job following the previous reviewer's suggestions.

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @david63011, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost4

 ## Highlights 💯 

- Good job following the previous reviewer's suggestions.   ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index2.html [hunk 4]
```diff
@@ -68,6 +67,7 @@ <h1 class="work-2">Works</h1>
           <h1 class="project-name1">Multi-Post Stories</h1>
           <h1 class="project-name-two">Gain+Glory</h1>
```
Comment (Abenezer-Tilahun) [inline index2.html:68]: - [ ] Good job for addressing the previous reviewer's suggestions using `section` tag wrap each part of your site, cover, projects, about, etc.

- [Optional ] kindly, I suggest to follow the previous reviewer's advice for accessibility , it is not recommended to use more than 1 H1 tag.Please fix it

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @david63011   , 

Great work on making the changes requested by a previous reviewer 👏
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Good commit messages

## Optional suggestions

- [Optional ] README file should be descriptive and  all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) should be present in your Repo.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by franclobo — COMMENTED** [review]
Hi <STUDENT_MENTION>, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has been already approved.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# INVALID ❌ ❌ ❌ 

Hi @david63011, 👋

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌
 - You have submitted a project, that has been already approved.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

✅ Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_⚠️ Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

----
_Invalid Code Review Request does not count into the code reviews limit._

**Pull request merged**